lib/Mojo/mysql.pm

```diff
@@ -251,6 +247,10 @@ easily.
   # Load migrations from file and migrate to latest version
   $mysql->migrations->from_file('/Users/sri/migrations.sql')->migrate;
 
+MySQL does not support nested transactions and DDL transactions.
+DDL statements cause implicit C<COMMIT>. C<ROLLBACK> will be called if
+migration script fails, but not everything will be reverted.
```

Comment:
What will be ROLLBACK? And what is "not everything"? I can accept a change, but then we need to point the reader to another resource (or give more details) on what is really going on.